src/types/IJitsiMeetingProps.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import IJitsiMeetingOptions from "./IJitsiMeetingOptions";
+import { JitsiMeeting } from "..";
+
+/**
+ * The type of the React {@code Component} props of {@link JitsiMeeting}.
+ */
+export default interface IJitsiMeetingProps {
+  /**
+   * The id of the component
+   */
+  id: string;
+  /**
+   * The domain used to build the conference URL
+   */
+  domain: string;
+
+  /**
+   * The tenant that provides an isolated context and prefixes the room name
+   */
+  tenant?: string;
```

Comment:
This doesn't apply to non jaas meetings.

Reply:
@saghul  we do have tenant notion outside of JaaS, e.g. WorkD users and 8x8 Meet users

Reply:
It's part of the room name really, not sure we want to expose the concept.

Reply:
we do expose it already one way or another on our products. I personally don't see a reason why not to expose it.

Reply:
Not opposed, but it was never part of the external API itself and it does require setup on the infra side. The Docker setup, for example, does not support tenants. Not all Jitsi installations support tenants. That's why I thought of not exposing it.